core/src/main/scala/filodb.core/query/QueryUtils.scala

```diff
@@ -0,0 +1,25 @@
+package filodb.core.query
+
+/**
+ * Storage for utility functions.
+ */
+object QueryUtils {
+  private val regexChars = Array('.', '?', '+', '*', '|', '{', '}', '[', ']', '(', ')', '"', '\\')
+  private val regexCharsMinusPipe = (regexChars.toSet - '|').toArray
+
+  /**
+   * Returns true iff the argument string contains any special regex chars.
+   */
+  def containsRegexChars(str: String): Boolean = {
```

Comment:
Wondering if it is a good idea to have a unit test just for the `containsRegexChars` and `isPipeOnlyRegex`. ( Please feel free to discard if we are already testing this path in unit tests )

Reply:
Good call -- I'll add a couple specifically for these methods.

Reply:
Thanks a tonne :)